spyderlib/utils/introspection/jedi_plugin.py

```diff
@@ -267,7 +266,7 @@ def preload(self):
     source_code = "import n"
     completions = p.get_completions(CodeInfo('completions', source_code,
         len(source_code)))
-    assert 'numpy' in completions
+    assert ('numpy', 'module') in completions
```

Comment:
We do not.  We have an "M" icon for method, but if we used that it would be the same one as the one used in the outline explorer.  I could make an "M" with a different color and use it for module.